### PR TITLE
Clean up code

### PR DESCRIPTION
**Pull request by inderdhir**

### CODE REVIEW

File: package/Runtime/Plugins/iOS/NimbusManager.swift [hunk 1]
```diff
@@ -50,7 +48,7 @@ import NimbusKit
     @objc public class func setGDPRConsentString(consent: String) {
         var user = NimbusRequestManager.user ?? NimbusUser()
         user.configureGdprConsent(didConsent: true, consentString: consent)
-        NimbusRequestManager.user = user
+        NimbusAdManager.user = user
```
Comment (inderdhir) [inline package/Runtime/Plugins/iOS/NimbusManager.swift:51]: Equivalent but ideally all our pubs always go through the NimbusAdManager

File: package/Runtime/Plugins/iOS/NimbusManager.swift [hunk 2]
```diff
@@ -106,29 +107,29 @@ import NimbusKit
         var impression = request.impressions[0]
         impression.video?.companionAds = [banner]
         request.impressions[0] = impression
-        
-        (Nimbus.shared.renderers[.forAuctionType(.video)] as? NimbusVideoAdRenderer)?.showMuteButton = false // false by default
```
Comment (inderdhir) [inline package/Runtime/Plugins/iOS/NimbusManager.swift:110]: false by default

**Review by marcsantiago — CHANGES_REQUESTED** [review]
No so much a change, but before merging you need to ensure everything builds in Unity and that all the functionality is the same, @brugg knows what to look for so work with him to ensure it's all working as intended. @brugg let me know when there is another build in test flight so that I can test these changes as well.

**Review by brugg — APPROVED** [review]

**Comment by brugg** [comment]
@marcsantiago the build was published to TestFlight. v0.0.3.

**Review by marcsantiago — APPROVED** [review]

**Comment by marcsantiago** [comment]
> @marcsantiago the build was published to TestFlight. v0.0.3.

Thanks @brugg downloaded and tested. Everything looks the same, this is good to go.

**Pull request merged**